### ISSUE TITLE
Feat: Transactions should be able to run inline in handlers

### ISSUE THREAD

**Issue by thantos** [functionless/eventual#448]

**Comment by sam-goodwin** [comment]
100%. Also, no need for them to be lambdas.

**Comment by sam-goodwin** [comment]
We need to make a decision: will transactions still be allowed to called directly over the API gateway? If so, we need to support middleware and zod schemas for auth and validation

**Comment by thantos** [comment]
I think it's be better if allow transactions to be run inline in non workflow handlers. Then wrap it in a command. Maybe add `command.transaction`

**Comment by sam-goodwin** [comment]
I agree! I think they should be entirely internal. They don't need to be named, schema'ed or anything. Just a utility.
